integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,27 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeUpperCaseProto(t *testing.T) {
+	name := "testbuildexpose"
```

Comment:
@tiborvass 
changed, please check.

BTW, I'll be on the trip soon.
If there are other comments, I had to do it in Thuesday or later.

Thanks